addons/binding/org.openhab.binding.synopanalyzer/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/SynopAnalizer.jar
```

Comment:
Can you please also add the about.html here it was just recently done for the others bindings.